packages/sanes-browser-extension/src/extension/background/model/persona/index.ts

```diff
@@ -294,6 +303,30 @@ export class Persona {
     return filtered;
   }
 
+  public async getBalances(): Promise<readonly Amount[]> {
```

Comment:
Can we make this return `Promise<ReadonlyArray<readonly Amount[]>>`, where each element represents one account and in each account there is a list of balances?

`(await getBalances())[0]` then holds the balances for the first account.

Reply:
I opened the Wallet's Balances tab to verify the tokens shown in the extension and it shows BASH, CASH, ETH, and LSK, whereas the extension only shows CASH. Is there something missing? Connections? Configuration in config files?

Reply:
Now they appeared on the extension without changing anything. I must be missing some faucet behaviour or something.